spec/abci/abci++_methods.md

```diff
@@ -417,11 +326,12 @@ title: Methods
 
 * **Usage**:
     * `RequestPrepareProposal`'s parameters `txs`, `misbehavior`, `height`, `time`,
-      `next_validators_hash`, and `proposer_address` are the same as in `RequestProcessProposal`.
+      `next_validators_hash`, and `proposer_address` are the same as in `RequestProcessProposal`
+      and `RequestFinalizeBlock`.
     * `RequestPrepareProposal.local_last_commit` is a set of the precommit votes that allowed the
-      decision of the previous block.
-    * Fields `height`, `time`, `proposer_address`, and `next_validators_hash` match the values from
-      the header of the proposed block.
+      decision of the previous block, together with their corresponding vote extensions.
+    * The `height`, `time`, and `proposer_address` values match the values from the header of the
```

Comment:
Hmm this is how it was in the 0.36 spec. I will check commits onto 0.37 to see whether the `next_validator_hash` info was added in that branch. I think it should be included in the sentence.

Reply:
OK, after some digging, I found out what's going on.
In #98, I was cherry-picking code that might have been forgotten in `v0.36.x`, that we might need. It was that PR that introduced `next_validators_hash`.
I didn't backport #98 to `v0.37.x` because the initial PR had to do with ABCI 2.0. Although I realize I should backport it. Will open a PR shortly.
Thanks for spotting @cason !

Reply:
#453

Reply:
I will merge this w/o this fix then ,since you opened a PR and then we can merge the PR into this.